Update 'Update your instance' (#345)
shlinkio/shlink.io: src/content/documentation/update-your-instance.mdx

However, the process may defer depending on how you installed Shlink.

<Callout type="warning" title="Important">
  It is recommended that you don't skip too many versions at a time when updating Shlink. If you have to, make sure to back-up your database beforehand, in case the process needs to be rolled back. 
  

>   If your Shlink instance is out of date, preferably, update Shlink in sequential order using the latest minor release for each update. For example, from version 3.2.1 to reach version 3.4.0, first update to 3.3.2 before updating to 3.4.0.

Reviewer: ```suggestion
  If your Shlink instance is out of date, preferably update Shlink in sequential order using the latest patch release for every minor version. For example, to go from version 3.2.1 to version 3.4.0, first update to 3.3.2, and then 3.4.0.
```
